docs/tutorial_bracket_strategy.md

```diff
@@ -0,0 +1,228 @@
+```
+
+Now, you should just follow the link and sign & execute the transaction with the lower transaction-ID first. It should look like this:
+
+![Read tokenAddressToIdMap](assets/tutorial_bracket_strategy_signing.png)
+
+For executing, just press the button _confirm_.
+
+If you are executing the second transaction with a lower gasPrice than the first one, then you can sign it immediately.
+Otherwise, you should wait until the first one is mined. This works in most cases, as the gas price usually determines the mining order of the transaction.
+
+Congrats! You just provided liquidity and have a chance to earn some passive income if the prices are moving and returning back to your initial `currentPrice`.
+
+## Withdrawing liquidity
+
+The withdrawal from the Gnosis Protocol is always done in two steps: Requesting and Claiming.
+
+### Withdraw request
+
+Withdrawal requests can be made with the following command:
+
+```ssh
+npx truffle exec scripts/withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma-separated-brackets]  --tokenIds=[indices] --requestWithdraw --network=$MASTER_SAFE
+```
+
+Here, the flag `--requestWithdraw` is very important part, which tells the script to initiate the the withdraw request.
+The `tokenIndices` of the tokens, which you want to withdraw from the brackets need to be specified via the flag `--tokenIds=[indices]` - do not write the brackets into the command.
+Under normal usage, these are the exact same indices, which were used during the liquidity provision.
+The flag `--brackets=[comma separated brackets]` must contain the brackets that were deployed during the liquidity provision. Please provide these addresses separated by commas and without any spaces.
+In case you don't remember them, you would have to run the following command:
+
+```ssh
+npx truffle exec scripts/get_deployed_brackets.js --masterSafe=$MASTER_SAFE --network=$NETWORK_NAME
+```
+
+which will display all your previously written brackets and generate a CSV listing the brackets with more information.
+
+By running the withdraw request script, you have generated a transaction within your Gnosis-safe interface. Execute this transaction and wait - maximal 5 minutes after the transaction gets mined - until the Gnosis Protocol closes the current auction batch.
+In the next batch you will be able to claim your funds.
+
+### Withdrawing funds
+
+The actual withdraw transaction transferring the tokens back into the MASTER_SAFE from the Gnosis Protocol can be initiated with the following command.
+
+```ssh
+npx truffle exec scripts/withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma-separated-brackets]  --tokenIds=[indices] --withdraw --transferFundsToMaster --network=$MASTER_SAFE
+```
+
+The parameters are essentially the same as in the requesting withdraw script, except for the new flag `--withdraw` instead of `--withdrawRequest`.
+
+## Even more secure setup with more owners
+
+(coming soon)
```

Comment:
Is this planned to be part of this PR or another one?

Reply:
I removed it. Let's first gauge the interest in the current tutorial